sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 1.6.0-beta.2 (Unreleased)
+## 1.6.0 (2024-04-09)
```

Comment:
Should add a new features section to this with what is being released new in this version from the beta releases